src/api/cli/integration/integration_test.go

```diff
@@ -41,7 +42,6 @@ var (
 )
 
 var (
-	update     = flag.Bool("update", false, "update golden files")
```

Comment:
Got varcheck error:
src/api/cli/integration/integration_test.go:45:2:warning: unused variable or constant update (varcheck)

Reply:
Can you undo this change? It was defined but not used yet https://github.com/skycoin/skycoin/pull/993, using it in this PR

Reply:
@therealssj , OK. I`ll return it and wait until #993 will be merged.